CMakeLists.txt

```diff
@@ -5,35 +5,119 @@ project(DDC VERSION 0.0.0 LANGUAGES CXX)
 
 # List of options
 
-option(BUILD_BENCHMARKS    "Build the benchmarks." OFF)
-option(BUILD_DOCUMENTATION "Build the documentation" OFF)
-option(DDC_BUILD_PDI_WRAPPER "Build the PDI wrapper" ON)
+option(BUILD_BENCHMARKS      "Build DDC benchmarks." OFF)
+option(BUILD_DOCUMENTATION   "Build DDC documentation/website" OFF)
+option(BUILD_EXAMPLES        "Build DDC examples" ON)
+option(DDC_BUILD_PDI_WRAPPER "Build DDC PDI wrapper" ON)
+
+
+# Dependencies
 
 set(DDC_DEPENDENCY_POLICIES "AUTO" "EMBEDDED" "INSTALLED" "SUBPROJECT")
 
-set(DDC_mdspan_DEPENDENCY_POLICY "AUTO" CACHE STRING "Policy to find the `mdspan` package. Options: ${DDC_DEPENDENCY_POLICIES}")
-set_property(CACHE DDC_mdspan_DEPENDENCY_POLICY PROPERTY STRINGS "${DDC_DEPENDENCY_POLICIES}")
 
-# Dependencies
+## CMake modules
+
+include(CMakePackageConfigHelpers)
+include(CTest)
+
+
+## OpenMP
+
+find_package(OpenMP REQUIRED)
 
-if(DDC_mdspan_DEPENDENCY_POLICY STREQUAL "AUTO")
+
+## mdspan
+
+set(DDC_mdspan_DEPENDENCY_POLICY "AUTO" CACHE STRING "Policy to find the `mdspan` package. Options: ${DDC_DEPENDENCY_POLICIES}")
+set_property(CACHE DDC_mdspan_DEPENDENCY_POLICY PROPERTY STRINGS "${DDC_DEPENDENCY_POLICIES}")
+if("${DDC_mdspan_DEPENDENCY_POLICY}" STREQUAL "AUTO")
```

Comment:
Why using `"${DDC_mdspan_DEPENDENCY_POLICY}"` instead of `DDC_mdspan_DEPENDENCY_POLICY` ?

Reply:
the second one is a leftover from before variables in CMake and depending on the name of the variable, can be an issue

Reply:
It could be an issue using `${DDC_mdspan_DEPENDENCY_POLICY}` but I guess `DDC_mdspan_DEPENDENCY_POLICY` is okay ? Reading the documentation it says if the variable does not exists it will be equivalent to `"DDC_mdspan_DEPENDENCY_POLICY" STREQUAL "AUTO"`, which seems as wrong as `"" STREQUAL "AUTO"`.

Reply:
I would recommend to never rely on the fact that some command will auto deref variables and always quote var usage so to avoid any ambiguity